### PR TITLE
ci(linter): enable several linters

### PR DESCRIPTION
**Pull request by mmorel-35**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -9,13 +11,19 @@ run:
 linters:
   fast: false
   enable:
-  - gofmt
```
Comment (arkodg) [inline .golangci.yml:12]: why is `gofmt` removed

File: .golangci.yml [hunk 2]
```diff
@@ -9,13 +11,19 @@ run:
...
-  - govet
   - gomodguard
+  - govet
   - misspell
-  - exportloopref
+  - revive
+  - unconvert
+  - unparam
+  - unused
+  - whitespace
```
Comment (arkodg) [inline .golangci.yml:26]: can we also add `gosec`
https://github.com/securego/gosec

File: .golangci.yml (hunk 1 above)
Comment (arkodg) [reply .golangci.yml:12]: sgtm, thanks for the clarification, linking the repo is case others are curious https://github.com/mvdan/gofumpt

File: .golangci.yml (hunk 2 above)
Comment (mmorel-35) [reply .golangci.yml:27]: It’s added !

**Review by arkodg — APPROVED** [review]

**Comment by robscott** [comment]
Thanks @mmorel-35!

/release-note-none
/lgtm
/approve

**Pull request merged**